aws/iam/datadog/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "datadog_aws_account" {
+  default     = "464622532012"
```

Comment:
should this value be in this repo?

Reply:
@stephen-puiszis yeah, it should be, but @phoolish I do not think it needs to be a variable. This does not change, and it is what DataDog publishes to the world. It isnt custom per account or anything